client/my-sites/plugins/constants.js

```diff
@@ -79,3 +79,33 @@ export const ECOMMERCE_BUNDLED_PLUGINS = [
 ];
 
 export const UNLISTED_PLUGINS = [ 'automated-db-schenker-shipping', 'wp-fusion-lite' ];
+
+export const WPBEGINNER_PLUGINS = [
+	'optinmonster',
+	'wpforms-lite',
+	'google-analytics-for-wordpress',
+	'all-in-one-seo-pack',
+	'coming-soon',
+	'wp-mail-smtp',
+	'custom-facebook-feed',
+	'duplicator',
```

Comment:
`duplicator` isn't compatible with wpcom yet. We await for AM to fix https://github.com/Automattic/jetpack/pull/39775 .

Reply:
Thanks! Removed it for now.